dataflow/src/main/java/org/checkerframework/dataflow/analysis/ConditionalTransferResult.java

```diff
@@ -32,31 +33,42 @@
      *
      * <p><em>Aliasing</em>: {@code thenStore} and {@code elseStore} are not allowed to be used
      * anywhere outside of this class (including use through aliases). Complete control over the
-     * objects is transfered to this class.
+     * objects is transferred to this class.
      *
+     * @param value the abstract value produced by the transfer function
+     * @param thenStore {@link #thenStore}
+     * @param elseStore {@link #elseStore}
+     * @param storeChanged {@link #storeChanged}
      * @see #ConditionalTransferResult(AbstractValue, Store, Store, Map, boolean)
      */
     public ConditionalTransferResult(
-            @Nullable A value, S thenStore, S elseStore, boolean storeChanged) {
+            @Nullable V value, S thenStore, S elseStore, boolean storeChanged) {
         this(value, thenStore, elseStore, null, storeChanged);
     }
 
     /**
-     * See {@link #ConditionalTransferResult(AbstractValue, Store, Store, Map, boolean)}.
+     * Create a new {@link #ConditionalTransferResult(AbstractValue, Store, Store)}.
```

Comment:
Should the linked constructor be consistent with the `@see` constructor below? At the moment this links to itself, which doesn't make much sense.
The implementation actually calls a constructor without a map...
Should it be documented that `false` is used for the boolean?
Same comment for the constructor below this one.